ansibullbot/triagers/plugins/shipit.py

```diff
@@ -261,6 +261,21 @@ def get_shipit_facts(issuewrapper, inmeta, module_indexer, core_team=[], botname
     ]
 
     files = [f for f in iw.files if not f.startswith(u'changelogs/fragments/')]
+
+    # https://github.com/ansible/ansibullbot/issues/1238
+    meta[u'component_matches'] = [
+        x for x in meta.get(u'component_matches', [])
+        if not x[u'repo_filename'].startswith(u'test/sanity')
+    ]
+    files = [f for f in files if not (f.startswith(u'test/sanity') and f.endswith(u'ignore.txt'))]
+
+    # make sure only deletions from ignore.txt are allowed
+    for pr_file in iw.pr_files:
+        if (pr_file.filename.startswith(u'test/sanity') and pr_file.filename.endswith(u'ignore.txt')):
+            if pr_file.additions > 0:
```

Comment:
Do we need a check for added files? The existing code checking for deletions has this:

https://github.com/ansible/ansibullbot/blob/a6e389c92b786e803df7c4bd88c6715257c065eb/ansibullbot/triagers/plugins/shipit.py#L114